jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -48,6 +48,16 @@
     jobs:
       - 'openstack-cleanvm-{release}'
 
+- project:
+    name: openstack-cleanvm-Train
+    disabled: false
+    release: Train
+    image:
+      - SLE_15
```

Comment:
Should we use SP1 here? (Same as Master)

Reply:
good point, fixed.